debian/control

```diff
@@ -0,0 +1,14 @@
+Source: aeternity-node
+Section: misc
+Maintainer: Aeternity Team <info@aeternity.com>
+Build-Depends: debhelper, erlang ( >=1:20.2-1),  erlang-base ( >=1:20.2-1), erlang-dev ( >=1:20.2-1),
```

Comment:
Could you prevent OTP 21? We are not really supporting it for now.

Reply:
Hm, I am not sure if this could be achieved. The issue will be with [building on 16.04](
https://github.com/aeternity/aeternity/blob/ubuntu_pacakging_162439771/docs/debian_ubuntu_packaging.md#ubuntu-1604-fixes)

The issues is that for Ubuntu/Debian package every dependency should come as a package. Otherwise a package cannot be created. So in 16.04, an  ErlangSolutions Debian repository is used to get Erlang with sufficient version as a package. 

However 18.04 provides this

```
apt-cache policy erlang
erlang:
  Installed: 1:20.2.2+dfsg-1ubuntu2
  Candidate: 1:20.2.2+dfsg-1ubuntu2
  Version table:
 *** 1:20.2.2+dfsg-1ubuntu2 500
        500 http://archive.ubuntu.com/ubuntu bionic/universe amd64 Packages
```

While the usable version as a package for 16.04 from the mentioned repository would be

```
apt-cache policy erlang | grep 20.2
  Candidate: 1:20.2-1
     1:20.2-1 501
```

There seems to be no way to have exact version in `Build-Depends`. I'll double-check the documentation, requirements and regular expressions supported in `Build-Depends`, but it either stays this way, or we drop 16.04.

Reply:
We can't ship a package with OTP 21 either

Reply:
Fixed in e737fba. Works as expected on 18.04 and 16.04 (with external repository)